CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This is a history of changes to clara-rules.
 * Fix issue with incorrect namespace qualification of rule and query code in ClojureScript. See [issue 359](https://github.com/cerner/clara-rules/issues/359).
 * Add clear-ns-productions! functionality to support clean reloading of rule and query definitions. See [issue 316](https://github.com/cerner/clara-rules/issues/316) for details.
 * Support session inspection and fact-graph in ClojureScript. See [issue 307](https://github.com/cerner/clara-rules/issues/307) for details.
+* Fix issue when compiling CLJS and session, rules, and fact-types are not in the same namespace or explicitly referred. See [issue 359](https://github.com/cerner/clara-rules/issues/359).
```

Comment:
Issue 359 is in the changelog already on line 5.  I believe I changed the changelog message when I did the release on that particular entry; basically I just wanted to specifically indicate what the issue was rather than just indicating that an issue existed.  If you think it could be more clear I'm open to making changes, we'll want to have a single bullet point in the changelog though.

Reply:
I just messed up the merge from upstream. Removing.